docs/baking.md

```diff
@@ -0,0 +1,185 @@
+systemd units for running the corresponding binaries in the background.
+
+### Downloading zcash-params
+
+Since v8.0 it's required to have the `sapling-output.params` and `sapling-spend.params` files from
+the `zcash-params` within the `XDG_DATA_DIRS` to run the `tezos-node`.
+
+We'll need these params to be accessible by different
+users (because `systemd` services are usually run as a separate user), thus the suggested way of getting
+them via [the zcash script](https://raw.githubusercontent.com/zcash/zcash/master/zcutil/fetch-params.sh), which will
+download them to the `HOME` directory of the current user, is not convenient. Moreover, it will
+download the large `sprout-groth16.params` which is unused at runtime.
+So, it's better to download them to the `/usr/local/share/zcash-params`, the easiest way to do that
+is to run [`fetch-sapling-params.sh`](../scripts/fetch-sapling-params.sh) script:
+```
+sudo ./fetch-sapling-params.sh
+```
+
+Sudo is required because the files are downloaded to the `/usr/local/share/zcash-params`.
+This script also checks that hashsums of the downloaded files match the expected values.
+
+## Setting up the tezos-node
+
+The easiest way to set up a node running on the `mainnet` or on one of the
+testnets (e.g. `delphinet` or `edonet`) is to use one of the predefined
+`tezos-node-<network>.services` systemd services provided in the `tezos-node`
+package. 
+
+However, by default, these services will start to bootstrap the node from scratch,
+which will take a significant amount of time.
+In order to avoid this, here we suggest bootstrapping from a snapshot instead.
+
+### Setting up node data directory
+
+`tezos-node-<network>.service` has `/var/lib/tezos/node-<network>` as a data directory
+and `http://localhost:8732` as its RPC address by default.
+These parameters can be changed in the service configuration, which is located in
+`/lib/systemd/system/tezos-node-<network>.service`. Note, that if you've updated the `.service`
+file, you should reload systemctl daemon via running `sudo systemctl daemon-reload`.
+
+### Bootstrapping the node
+
+In order to run a baker locally, you'll need a fully-synced local `tezos-node`.
+
+The fastest way to bootstrap the node is to import a snapshot.
+Snapshots can be downloaded from https://snapshots.tulip.tools/#/ .
+
+All commands within the service are run under the `tezos` user.
+
+Download the snapshost for desired network and with the desired history mode
+(you can read more about `tezos-node` history modes [here](https://tezos.gitlab.io/user/history_modes.html#history-modes)).
+
+Snapshot import requires that the `tezos-node` data directory doesn't contain the `context` and `store` folders.
+Remove them before the snapshot import.
+
+In order to import the snapshot run the following command:
+```
+sudo -u tezos-node snapshot import --data-dir /var/lib/tezos/node-<network>/ <path to the snapshot file>
```

Comment:
```suggestion
sudo -u tezos tezos-node snapshot import --data-dir /var/lib/tezos/node-<network>/ <path to the snapshot file>
```
missing the user :smile: 

Also, I'm trying following this on a fresh VM and it seems that there is no `tezos` user at this point, should it have been created by some of the previous steps (I doubt it, not much has happened), by one of the following ones, or should it be created manually?
If the latter is the case I think just above where we mention that all commands are run under `tezos` it should also say how to create the user.

Reply:
There's another problem at this point, if one has never `start`ed the node service with `systemctl` before there in no `/var/lib/tezos` directory and this command will fail (with an `Invalid data directory` error which is not very helpful).

So I suggest that we either:
1. add the commands to create the data directory manually in the `Setting up node data directory` section
2. rearrange this part of the guide a bit and have the user `start` the node in that section and the `stop` it and `import` the snapshot after.

What do you think?

P.s. starting the `node` seems to not create the `tezos` user automatically, unless I've done something wrong

Reply:
`/var/lib/tezos` should be created as part of .deb installation

Reply:
I assumed that both `tezos` user and `/var/lib/tezos` are both created during the package installation, but looks like it's not true :confused:. 
I Will check that additionally

Reply:
The aforementioned problems with user and directory creations should be fixed now